apps/protocol-frontend/src/components/DaoDashboardShell.tsx

```diff
@@ -1,29 +1,105 @@
-import { Container, Box, Stack, Text } from '@chakra-ui/react';
-import SiteLayout from '../components/SiteLayout';
-import { GOVRN_MOTTO } from '../utils/constants';
+import { Box, Flex, Heading } from '@chakra-ui/react';
+import PageHeading from '../components/PageHeading';
+interface DaoDashboardShellProps {
+  daoName: string;
+}
 
-const DaoDashboardShell = () => (
-  <SiteLayout>
-    <Container
+const DaoDashboardShell = ({ daoName }: DaoDashboardShellProps) => {
+  return (
+    <Box
       paddingY={{ base: '4', md: '8' }}
-      paddingX={{ base: '0', md: '8' }}
+      paddingX={{ base: '4', md: '8' }}
       color="gray.700"
-      maxWidth="1200px"
+      width="100%"
     >
-      <Box
+      <PageHeading>{daoName}</PageHeading>
+      <Flex
+        justify="space-between"
+        direction="column"
+        wrap="wrap"
+        width="100%"
+        paddingX={4}
+        paddingY={4}
         background="white"
         boxShadow="sm"
         borderRadius={{ base: 'none', md: 'lg' }}
       >
-        <Stack spacing="4" justify="center" align="center" minHeight="50vh">
-          <Text>{GOVRN_MOTTO}</Text>
-          <Text fontSize="lg" fontWeight="medium">
-            DAO Dashboard
-          </Text>
-        </Stack>
-      </Box>
-    </Container>
-  </SiteLayout>
-);
+        <Flex direction="column" gap={4}>
+          <Flex direction={{ base: 'column' }} gap={2}>
+            <Flex
+              direction={{ base: 'column', lg: 'row' }}
+              gap={8}
+              justifyContent="space-apart"
+            >
+              {/* <Flex flexBasis="50%"> */}
+              {/* DatePicker will go here */}
+              {/* </Flex> */}
```

Comment:
Extra commented code

Reply:
Will remove in next PR - no need to have this in since it's only indicating where the component will go. Thanks!